content/en/docs/concepts/services-networking/service.md

```diff
@@ -887,30 +887,30 @@ There are other annotations for managing Cloud Load Balancers on TKE as shown be
     metadata:
       name: my-service
       annotations:
-        # Bind Loadbalancers with speicfied nodes
+        # Bind Loadbalancers with specified nodes
```

Comment:
Using `git add -p` or equivalent would mean you could propose just the relevant change. If you can do that @jmyung, please do.

Some IDEs that talk Git can also let you selectively commit changes.

Reply:
Thank you for the tip. @sftim 🎅 
I applied with `git app -p`